test/Norm/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 file(GLOB cppfiles *.cpp)
 add_executable(norm_test ${cppfiles})
 set_khiops_options(norm_test)
-target_link_libraries(norm_test GTest::gtest_main base)
+target_link_libraries(norm_test GTest::gtest_main base testutils)
```

Comment:
Should this be done only if `TESTING=ON`?

Reply:
This CMakeLists is added only if TESTING=ON

Reply:
Ok